### PR TITLE
Fixed Linux URL

### PR DESCRIPTION
**Pull request by ianblenke**
The URL path pattern has changed for Linux

### CODE REVIEW

File: fetch.sh [hunk 1]
```diff
@@ -39,7 +45,10 @@ QCOW2="$(basename "$TMP_DIR"/*.ovf .ovf).qcow2"
 qemu-img convert -f vmdk -O qcow2 "$VMDK" "$QCOW2"
 
 # Remove now-useless files
-rm "$VMDK" "$TMP_DIR"/*.ovf
+rm "$VMDK" "$TMP_DIR"/*.ov[fa]
 
-echo Finished! Delete "$TMP_DIR" to tidy up
+echo Finished! Deleting "$TMP_DIR" to tidy up
+ls -la "$TMP_DIR"
+rm -fr "$TMP_DIR"
```
Comment (lentinj) [inline fetch.sh]: I didn't do this on purpose. As hinted in the docs, I like to tweak the VM image a bit (get IE to launch on startup, set homepage), once I'm satsfied I've not done something stupid, then I remove the TMP_DIR. Or just get around to removing it when I'm short on space :)

Secondly, ``rm -rf`` isn't something anyone likes to see in scripts, and doesn't seem necessary off the top of my head.

File: fetch.sh [hunk 2]
```diff
@@ -39,7 +45,10 @@ QCOW2="$(basename "$TMP_DIR"/*.ovf .ovf).qcow2"
...
+ln -sf "$QCOW2" "disk.qcow2"
```
Comment (lentinj) [inline fetch.sh]: Is this to have a "default" image? Most people using this I'm guessing will be downloading multiple versions, so this symlink will just fight between whatever you downloaded.

What we could do though is make start.sh search for the most recent qcow2 image if none is given, which would be slightly neater.

**Comment by lentinj** [comment]
Thanks for this! I didn't realise the format had changed and my IE VMs are starting to expire, so need re-downloading. Good timing :)

I'm somewhat nervous of the various ``rm -f``'s in the changes though. I'll have a go at neatening them later today and see what you think.

File: fetch.sh (hunk 1 above)
Comment (lentinj) [inline fetch.sh]: Could make it an option on the script though, --keep or something.

**Comment by lentinj** [comment]
Have a look at the new-zip-format branch. I think this works, and avoids having to make more temporary files. A bit more testing, and will merge this into master.

I'm guessing this pull request including the VNC option was a mistake? Probably not something to enable by defualt for the world.

**Comment by ianblenke** [comment]
Including VNC by default was intentional mostly because I'm deploying this on CoreOS hosts on secured VPC segments - it would make sense to bind to 127.0.0.1 though. Good advice!

**Pull request closed without merging**